store/src/java/com/zimbra/cs/index/DBQueryOperation.java

```diff
@@ -445,7 +444,7 @@ public void addTag(Tag tag, boolean bool) {
 
     @Override
     public void close() {
-        Closeables.closeQuietly(luceneOp);
+        luceneOp.close();
```

Comment:
This might need some safety code round it.  Explore callers.  Can luceneOp ever be null?